### PR TITLE
Implementation of ivy.functional.frontend.paddel.tensor.argmin(#16282)

### PR DESCRIPTION
**Pull request by JagjeetSunghera**

### CODE REVIEW

**Review by JagjeetSunghera — COMMENTED** [review]
exception .idea is not ingnored by gitingnore

**Review by JagjeetSunghera — COMMENTED** [review]
reviewed for any syntax error. found none

**Comment by JagjeetSunghera** [comment]
@xoiga123 please provide review over this branch.

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -125,6 +125,10 @@ def sinh(self, name=None):
     def argmax(self, axis=None, keepdim=False, dtype=None, name=None):
         return ivy.argmax(self._ivy_array, axis=axis, keepdims=keepdim, dtype=dtype)
 
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+    def argmin(self, axis=None, keepdim=False, dtype=None, name=None):
```
Comment (xoiga123) [inline ivy/functional/frontends/paddle/tensor/tensor.py:129]: Please implement the exact same signature of [argmin](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/argmin_en.html).

**Review by xoiga123 — CHANGES_REQUESTED** [review]
Thank you for contributing to Ivy :hugs: Overall looks good to me, but please revert the change made to `.idea/` files :back: Thank you!

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (JagjeetSunghera) [reply ivy/functional/frontends/paddle/tensor/tensor.py:129]: made a small mistake i will resolve it into a new PR

**Pull request closed without merging**